adapters/akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala

```diff
@@ -82,8 +85,9 @@ trait AkkaHttpAdapter {
 
   def makeWebSocketService[R, E](
     interpreter: GraphQLInterpreter[R, E],
-    skipValidation: Boolean = false
-  )(implicit ec: ExecutionContext, runtime: Runtime[R], materializer: Materializer): Route = {
+    skipValidation: Boolean = false,
+    keepAliveTime: Option[Duration] = None
+  )(implicit ec: ExecutionContext, runtime: Runtime[R with Clock with Random], materializer: Materializer): Route = {
```

Comment:
No need for `Random` I guess?

Reply:
You do, otherwise you can't add jittered to the timing

Reply:
Oh, I see. But why using jittered at all? You expect the keep-alive to be regular no?

Reply:
Not really, adding jitter help prevent congestion, it's not part of the spec per-se, but it's common in other keep-alive mechanisms I've seen elsewhere.